packages/renderer/noteStyle.ts

```diff
@@ -58,6 +59,9 @@ export default function(theme: any, options: Options = null) {
 
 	const fontFamily = '\'Avenir\', \'Arial\', sans-serif';
 
+	const isMobile = platformUtil.isMobile();
```

Comment:
Consider adding an option to `Options` rather than moving `platformUtil` to `utils`. Because rendering happens within a WebView (see e92f89df99de87d980a0622187d97bc67714aee1), `platformUtil.isMobile` returns `false` on mobile platforms.

Reply:
Fixed